### DISCUSSION TITLE
OAuth not being redirected to an additional redirect URL

### DISCUSSION THREAD

**Discussion by tessamerrill** [supabase/supabase#5360] [Questions]
I have an additional redirect URL set in my Auth settings, and after I authenticate through Supabase, it only redirects back to the base URL. 

My code:
```
supabase.auth.signIn({ provider: 'discord' }, { redirectTo: \`${process.env['NEXT_PUBLIC_AUTH_ENDPOINT'] || https://dash.example.com'}/auth/callback` })
```

The environment variable is set, and if it isn't to have a fallback url. Neither of them work and it still redirects to the base URL.

Am I doing something wrong?

Thanks!

**Comment by Fullbrains** [comment]
I have the same issue.

**Reply by fadethegap** [reply]
You might try this:

```
await supabase.auth.signIn(
        {
          provider:'discord',
        },
        { redirectTo: 'http://localhost:3000/route' }
      )
```

I have found that it requires the entire URL in order to work. Of course you would replace the hard coded URL with an environment variable.
